### DISCUSSION TITLE
How to make users just can select datetime in a single day?

### DISCUSSION THREAD

**Discussion by TienNg21** [nhn/tui.calendar#1297] [Q&A]
## Summary
When I use the calendar with view  **month** and **week**, I just want to create events in a single day. Then I change the start time and the end time in events, but my user can select datetime in more than one day. 

How to make users just can select datetime in a single day?  Any Ideas for this problem?

P/s: So sorry if my English make you confuse.

## Version
2.1.0

## Current Behaviour
[Screencast from 10-08-2022 13:57:46.webm](https://user-images.githubusercontent.com/62454445/183835590-d39ca679-1375-4305-ad5f-a50db176d639.webm)

## Expected Behaviour

I want to create events like Google Calendar

[Screencast from 10-08-2022 13:59:00.webm](https://user-images.githubusercontent.com/62454445/183835787-dd37af45-c884-41a2-85e1-af279032795b.webm)

**Comment by adhrinae** [comment]
@TienNg21 

It is intended behavior and not customizable by options. It's similar to the Apple calendar's interaction.

You can try the previous major version for your use case. It works exactly like what you wanted.

https://www.npmjs.com/package/tui-calendar
